Source/UserManagement/Read/StaffUsers/UserEventProcessor.cs

```diff
@@ -102,10 +102,6 @@ public UserEventProcessor(IStaffUsers collection)
         public void Process(PhoneNumberRegistered @event)
         {
             var baseUser = _collection.GetById<BaseUser>(@event.StaffUserId);
-            if (baseUser == null)
```

Comment:
Excellent!  Great that you remove these - we have to assume that validation and/or business rules on the Command side was done correctly and that this code can naively carry on.